pyam/utils.py

```diff
@@ -170,8 +170,11 @@ def read_pandas(path, sheet_name="data*", *args, **kwargs):
         # remove unnamed and empty columns, and rows were all values are nan
         def is_empty(name, s):
             if str(name).startswith("Unnamed: "):
-                if len(s) == 0 or all(np.isnan(s)):
-                    return True
+                try:
+                    if len(s) == 0 or all(np.isnan(s)):
+                        return True
```

Comment:
According to codecov this line is not tested. Not sure it this one line really needs a unit test though.

Reply:
Ok then I would say it's fine to leave it in as a safeguard against corrupted files.